docs/en/sampling_params.md

```diff
@@ -52,6 +55,26 @@ spaces_between_special_tokens: bool = True,
 regex: Optional[str] = None,
 # Do parallel sampling and return `n` outputs.
 n: int = 1,
+
+## Penalties. See [Performance Implications on Penalties] section below for more informations.
+
+# Float that penalizes new tokens based on their frequency in the generated text so far.
+# Values > 0 encourage the model to use new tokens, while values < 0 encourage the model to
+# repeat tokens.
+frequency_penalty: float = 0.0,
```

Comment:
https://platform.openai.com/docs/api-reference/chat/create#chat-create-frequency_penalty

Reply:
The official OpenAI documentation supports a range from -2 to 2, each value having its specific usage and meaning.
> Reasonable values for the penalty coefficients are around 0.1 to 1 if the aim is to just reduce repetitive samples somewhat. If the aim is to strongly suppress repetition, then one can increase the coefficients up to 2, but this can noticeably degrade the quality of samples. Negative values can be used to increase the likelihood of repetition.
Should we consider aligning with OpenAI?

Reply:
They are already enforced to be in the range, same as OpenAI spec. I will mention about that enforcement to this doc as well. I just mostly copy-pasted from [what it is in vLLM](https://github.com/vllm-project/vllm/blob/main/vllm/sampling_params.py#L49-L60)

Reply:
We may include some negative value test cases in the unit tests.